src/main/java/school/hei/haapi/model/Course.java

```diff
@@ -34,6 +44,8 @@ public class Course implements Serializable {
 
   private Integer totalHours;
 
+  private boolean isDeleted = false;
```

Comment:
why = false !

Reply:
you said that we need to set it to false by default